onboarding/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:dist="http://schemas.android.com/apk/distribution"
+    package="com.hedvig.onboarding">
+
+    <dist:module
+        dist:instant="false"
+        dist:onDemand="false"
+        dist:title="@string/title_onboarding">
+        <dist:fusing dist:include="false" />
+    </dist:module>
+
+    <application>
+
+        <activity
+            android:name=".moreoptions.MoreOptionsActivity"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+            android:launchMode="singleTask"
+            android:theme="@style/Hedvig.Theme"
+            android:windowSoftInputMode="adjustResize" />
+
+        <activity
+            android:name="com.hedvig.onboarding.chooseplan.ChoosePlanActivity"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+            android:launchMode="singleTask"
+            android:theme="@style/Hedvig.Theme"
+            android:windowSoftInputMode="adjustResize">
+
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="http" />
+                <data
+                    android:host="instantapptest.dev.hedvigit.com"
```

Comment:
Probably need to disable these 🙂

Reply:
We are still using these for navigation, like so:

```
private const val DEFAULT_URL = "https://instantapptest.dev.hedvigit.com"

fun Activity.startActivityWithAppLink(appLink: AppLink) {
    val uri = getUri(appLink)
    val intent = Intent(Intent.ACTION_VIEW, uri)
    intent.addCategory(Intent.CATEGORY_BROWSABLE)
    startActivity(intent)
}

private fun getUri(appLink: AppLink): Uri {
    return appLink.query?.let {
        Uri.parse("$DEFAULT_URL/${appLink.path}?${it.first}=${it.second}")
    } ?: Uri.parse("$DEFAULT_URL/${appLink.path}")
}
```

I don't think we will be able to open these activities externally anyway since the appLinks.json isnt set up, or am I missing something?

Reply:
We won't be able to open them externally when the app is not installed, but we will do so when it is installed.

Perhaps we can transition them to be on a proper hedvig.com-domain?